components/webgpu/wgpu_thread.rs

```diff
@@ -1232,6 +1229,7 @@ impl WGPU {
                             .copy_from_slice(&array_buffer);
                         }
                         let result = gfx_select!(buffer_id => global.buffer_unmap(buffer_id));
+                        // TODO: All errors should be silenced here
```

Comment:
I have yet to do this, relevant bug in FF: https://bugzilla.mozilla.org/show_bug.cgi?id=1842297

Reply:
DONE!